### ISSUE TITLE
HCM Testing

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#4178]
Go through all of our components in Windows High Contrast Mode and check for improvements. Go through the checklist on default settings (background: black, text: white, active: yellow, disabled: green, focus: teal) as well as light settings (https://www.w3.org/TR/css-color-4/#css-system-colors).

### Checklist per component

- [ ] Text is readable
- [ ] Focus is visible
- [ ] Icons are visible
- [ ] Borders are visible
- [ ] Colours are used correctly
- [ ] No elements are visually disturbing the order of importance (falling out of line)
